File: Ekklesia.Spa/src/app/models/FilterRule.ts
import { FilterEnum } from "./FilterEnum";

export class FilterRule {
    Type: FilterEnum;
    Field: string
    Arg: string

    constructor(field: string = '', arg: string = '', type: FilterEnum = FilterEnum.Equal) {
        this.Field = field
        this.Arg = arg
        this.Type = type
    }
}

File: Ekklesia.Domain/Filters/FilterRule.cs
namespace Ekklesia.Domain.Filters
{
    public class FilterRule
    {
        public FilterType Type { get; set; }
        public string Field { get; set; }
        public string Arg { get; set; }

        public FilterRule()
        {
            this.Field = string.Empty;
            this.Arg = string.Empty;
        }
        
    }
}
